pkg/services/container/morph/executor.go

```diff
@@ -361,7 +361,7 @@ func (s *morphExecutor) validateToken(t *sessionV2.Token, cIDV2 *refs.ContainerI
 
 	err = keyFromToken.Decode(t.GetSignature().GetKey())
 	if err != nil {
-		return errors.New("error while decoding public key from the token's signer")
+		return fmt.Errorf("error while decoding public key from the token's signer: %w", err)
```

Comment:
`decoding key from signer:`? It's rather obvious that it's an error and likely we know that it happens in token, messages better be concise.

Reply:
> we know that it happens in token

hehehehe (https://github.com/nspcc-dev/neofs-node/issues/2744)

> decoding key from signer:?

don't mind, fixed